scss/_base_links.scss

```diff
@@ -5,9 +5,18 @@
     text-decoration: none;
     border-bottom: 1px solid;
     font-weight: bold;
+    transition: all .5s;
 
     &:focus {
-      outline: thin dotted;
+      outline: thin dotted $color-warm-grey;
+    }
+
+    &:hover {
+      border-bottom: 1px solid $color-warm-grey;
```

Comment:
I would suggest modifying the`border-bottom-color` her instead as it would override and theme changes.